.github/workflows/md_lint_spell.yml

```diff
@@ -17,8 +17,8 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
 
-    - name: Lint Markdown files
-      uses: nosborn/github-action-markdown-cli@v3.3.0
+    - name: MD Link Check
+      uses: ngaurav-nelson/github-action-markdown-link-check@v1
       with:
         files: 'content/features/*.md'
```

Comment:
For this particular action, the parameter for the path is `folder-path`

https://github.com/gaurav-nelson/github-action-markdown-link-check#custom-variables